publishers/communities/sig-cli/render-helm-chart/krm-function-metadata.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: Add CI to validate the metadata here.
+
+apiVersion: config.kubernetes.io/v1alpha1
+kind: KRMFunctionDefinition
```

Comment:
This is based on what the KrmFunctionDefinition spec will look like after https://github.com/kubernetes/enhancements/pull/3220 goes in